### PR TITLE
add citation information to the readme

### LINKED ISSUE #724: Add citation information to Readme.
https://ieeexplore.ieee.org/document/8671560

### PR DESCRIPTION
**Pull request by bcumming**
Fixes #724

### CODE REVIEW

**Comment by bcumming** [comment]
I have made the changes requested by Anne, most notably adding citation information to the ReadTheDocs documentation.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ Arbor is a library for implementing performance portable network simulations of
...
+* Version 0.1: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1459679.svg)](https://doi.org/10.5281/zenodo.1459679)
+
+The following BibTeX can be used to cite Arbor:
+
+```
+@INPROCEEDINGS{
+    paper:arbor2019,
+    author={N. A. {Akar} and B. {Cumming} and V. {Karakasis} and A. {Küsters} and W. {Klijn} and A. {Peyser} and S. {Yates}},
+    booktitle={2019 27th Euromicro International Conference on Parallel, Distributed and Network-Based Processing (PDP)},
+    title={{Arbor -- A Morphologically-Detailed Neural Network Simulation Library for Contemporary High-Performance Computing Architectures}},
```
Comment (halfflat) [inline README.md]: The en-dash `--` should be an em-dash `---`.

File: README.md [hunk 2]
```diff
@@ -7,3 +7,25 @@ Arbor is a library for implementing performance portable network simulations of
...
+    year={2019}, month={Feb}, volume={}, number={},
```
Comment (halfflat) [inline README.md]: `month` should be either `feb` (without the brackets) or `2` (this is for portability: month names like `feb` are essentially macros that expand to a locale-specific string.)

File: README.md [hunk 3]
```diff
@@ -7,3 +7,25 @@ Arbor is a library for implementing performance portable network simulations of
...
+    pages={274-282},
```
Comment (halfflat) [inline README.md]: `-` should be `--` for an en-dash

File: doc/index.rst [hunk 4]
```diff
@@ -38,6 +38,36 @@ Some key features include:
...
 
+Citing Arbor
+------------
+
+.. |DOI-v0.1| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.2583709.svg
+    :target: https://doi.org/10.5281/zenodo.2583709
+
+.. |DOI-v0.2| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1459679.svg
+     :target: https://doi.org/10.5281/zenodo.1459679
+
```
Comment (halfflat) [inline doc/index.rst]: The DOIs and links are the wrong way around: first one is the v0.2 link, the second, the v0.1 link.

File: doc/index.rst [hunk 5]
```diff
@@ -38,6 +38,36 @@ Some key features include:
...
+    @INPROCEEDINGS{
+        paper:arbor2019,
+        author={N. A. {Akar} and B. {Cumming} and V. {Karakasis} and A. {Küsters} and W. {Klijn} and A. {Peyser} and S. {Yates}},
+        booktitle={2019 27th Euromicro International Conference on Parallel, Distributed and Network-Based Processing (PDP)},
+        title={{Arbor -- A Morphologically-Detailed Neural Network Simulation Library for Contemporary High-Performance Computing Architectures}},
+        year={2019}, month={Feb}, volume={}, number={},
+        pages={274-282},
+        doi={10.1109/EMPDP.2019.8671560},
+        ISSN={2377-5750}}
+
```
Comment (halfflat) [inline doc/index.rst]: Same edits as for the README.md

**Pull request merged**